bio/spades/metaspades/test/Snakefile

```
@@ -0,0 +1,34 @@
+container: "docker://continuumio/miniconda3:4.4.10"
+
+
+rule run_metaspades:
+    input:
+        reads=["test_reads/sample1_R1.fastq.gz", "test_reads/ample1_R2.fastq.gz"],
+    output:
+        contigs="assembly/contigs.fasta",
+        scaffolds="assembly/scaffolds.fasta",
+        dir=directory("assembly/intermediate_files"),
+    benchmark:
+        "logs/benchmarks/assembly/spades.txt"
+    params:
+        # all parameters are optional
+        k="auto",
+        extra="--only-assembler",
+    log:
+        "log/spades.log",  #stdout/stderr is written in first log file
+    threads: 8
+    resources:
+        mem_mem=250000,
+        time=60 * 24,
+    wrapper:
+        "master/bio/bwa/aln"
```

This is likely a relict from the wrapper you used as a template. If you add a testcase, this should throw an error.